packages/sandcastle/gallery/imagery-assets-available-from-ion/main.js

```diff
@@ -22,10 +22,9 @@ const dropdownOptions = [
   { label: "Google Maps 2D Roadmap", assetId: 3830184 },
   { label: "Google Maps 2D Satellite", assetId: 3830182 },
   { label: "Google Maps 2D Satellite with Labels", assetId: 3830183 },
-  { label: "Bing Maps Aerial", assetId: 2 },
-  { label: "Bing Maps Aerial with Labels", assetId: 3 },
-  { label: "Bing Maps Road", assetId: 4 },
-  { label: "Bing Maps Labels Only", assetId: 2411391 },
+  { label: "Azure Maps Aerial", assetId: 3891168 },
+  { label: "Azure Maps Roads", assetId: 3891169 },
```

Comment:
Is there no Aerial with Labels option anymore?

Reply:
yes just these 3 options for Azure https://github.com/CesiumGS/cesium-ion/issues/8265#issuecomment-3391955342 A user could add the satellite layer and labels layer on top for the same effect.